src/Integration/Monolog/CloudWatch/composer.json

```diff
@@ -11,9 +11,9 @@
         "Logs"
     ],
     "require": {
-        "php": "^7.2.5 || ^8.0",
+        "php": "^8.0",
```

Comment:
please revert.

Instead of using UnionType, you can either:

a. Remove the typehint from the method, and perform the check at runtime
```
/**
 * @param LogRecord|array $record
 */
protected function write($record): void
  if($record instanceof LogRecord){
    $formatted = $record->formatted;
    $record = array_merge($record->toArray(), ['formatted'=>$formatted]);
  }
  if (!is_array($record)) {
    throw new \TypeError(sprintf('Argument 1 passed to &s must be of the type array or LogRecord, %s given', __METHOD__, get_debug_type($res)));
  }

  // ...
}
```

b. or Use dynamic trait to provide both implementations

similar to https://github.com/symfony/symfony/blob/6.1/src/Symfony/Bridge/Monolog/Handler/CompatibilityHandler.php